python/addPFCands_cff.py

```diff
@@ -62,6 +62,8 @@ def addPFCands(process, runOnMC=False, allPF = False, onlyAK4=False, onlyAK8=Fal
                                                         idx_name = cms.string("pFCandsIdx"),
                                                         nameSV = cms.string("FatJetSVs"),
                                                         idx_nameSV = cms.string("sVIdx"),
+																												nameMu = cms.string("FatJetMuons"),
```

Comment:
[Style] Indentation level seems to vary for the newly added lines, compared to the surrounding statements
Especially here, this looked like an empty line and caused some confusion first